project.clj

```diff
@@ -4,9 +4,12 @@
   :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
             :url  "https://www.eclipse.org/legal/epl-2.0/"}
   :dependencies [[org.clojure/clojure "1.10.0" :scope "provided"]
-                 [org.slf4j/slf4j-log4j12 "1.7.32"]
+                 [com.fzakaria/slf4j-timbre "0.3.21"]
                  [com.taoensso/timbre "5.1.0"]
```

Comment:
Let's bump timbre to 5.2.1 as well?

Reply:
Fixed in aceb620368f68b8b44671ba7af9cf2ea80369792